yarn.lock

```diff
@@ -19744,6 +19751,7 @@ __metadata:
     "@grafana/e2e-selectors": "workspace:*"
     "@grafana/eslint-config": 2.5.1
     "@grafana/experimental": 0.0.2-canary.11
+    "@grafana/google-sdk": ^0.0.2
```

Comment:
Even though minor releases should not include breaking changes, it's probably safer to remove the `^` and pin to version 0.0.2 so we can control upgrades manually.

Reply:
FTR, 0.x versions behave differently than "stable" versions, for example, `^0.0.2` does not match 0.0.3: https://jubianchi.github.io/semver-check/#/^0.0.2/0.0.3 but yeah, there is no point of using `^0.0.2` since it's mostly the same than 0.0.2.